code/__HELPERS/roundend.dm

```diff
@@ -457,6 +459,31 @@
 			parts += G.get_result()
 		return "<div class='panel stationborder'><ul>[parts.Join()]</ul></div>"
 
+///Generate a report for how much money is on station, as well as the richest crewmember on the station.
+/datum/controller/subsystem/ticker/proc/market_report()
+	var/list/parts = list()
+	parts += "<span class='header'>Station Economic Summary:</span>"
+	///This is the richest account on station at roundend.
+	var/datum/bank_account/mr_moneybags
+	///This is the station's total wealth at the end of the round.
+	var/station_vault = 0
+	for(var/i in SSeconomy.bank_accounts)
+		if(istype(i, /datum/bank_account/department) || istype(i, /datum/bank_account/remote))
+			continue
+		var/datum/bank_account/current_acc = i
+		station_vault += current_acc.account_balance
+		if(!mr_moneybags)
+			mr_moneybags = current_acc
+		if(mr_moneybags.account_balance < current_acc.account_balance)
+			mr_moneybags = current_acc
```

Comment:
if u change mr_moneybags to the current account then the account balance will always be the same (theerefore not lesser than) the current account balance therefore you just need one line brother

```suggestion
		if(!mr_moneybags || mr_moneybags.account_balance < current_acc.account_balance)
			mr_moneybags = current_acc
```